test/cypress/integration/group-payments.spec.js

```diff
@@ -131,7 +131,7 @@ describe('Payments', () => {
     // After closing the modal it should dislay how much user need
     cy.getByDT('headerNeed').should('contain', 'You need $100')
     // The user should be inform that even if he can\'t pledge he can still contribute
-    cy.getByDT('givingParagraph').should('contain', 'You can contribute to your group with money or other valuables like teaching skills, sharing your time ot help someone. The sky is the limit!')
+    cy.getByDT('givingParagraph').should('contain', 'You can contribute to your group with money or other valuables like teaching skills, sharing your time to help someone. The sky is the limit!')
```

Comment:
Re tests: do you think you can add a couple tests to test that this new widget works?

Reply:
Yes, it is hooked up to the store! The payment progress is the only part not hooked yet. But nevertheless, I just added some tests to Contributions Widget as part of `group-payments.spec.js`!